modules/javafx.graphics/src/test/java/test/javafx/scene/layout/AnchorPaneTest.java

```diff
@@ -25,7 +25,14 @@
 
 package test.javafx.scene.layout;
 
-import test.javafx.scene.layout.MockBiased;
+import javafx.beans.property.DoubleProperty;
+import javafx.beans.property.SimpleDoubleProperty;
+import javafx.geometry.Bounds;
+import javafx.scene.Scene;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
```

Comment:
In our tests, we don't mix junit4 and junit5 APIs in single test class.
Could you please find an alternate/junit4 way for these two.

Reply:
There is not really an alternative for this in junit4. I can create a new test class instead.

Reply:
There's no need to do that. `AnchorPaneTest` works just as well if you simply use the JUnit5 imports exclusively.

Reply:
good point, that will be the better solution.